src/main/java/com/epam/reportportal/junit/ReportPortalListener.java

```diff
@@ -487,17 +489,22 @@ protected void reportSkippedStep(@Nonnull final Object runner, @Nonnull final Fr
 		TestItemTree.TestItemLeaf testLeaf = ofNullable(getLeaf(runner)).orElseGet(() -> context.getItemTree()
 				.getTestItems()
 				.get(myParentKey));
-		AtomicTest<FrameworkMethod> testContext = LifecycleHooks.getAtomicTestOf(runner);
-		Description description = testContext.getDescription();
-		FrameworkMethod method = testContext.getIdentity();
-		ofNullable(testLeaf).ifPresent(l -> {
-			StartTestItemRQ startRq = buildStartStepRq(runner, description, method, callable, skipStartTime);
-			Maybe<String> id = launch.get().startTestItem(l.getItemId(), startRq);
-			ofNullable(throwable).ifPresent(t -> sendReportPortalMsg(id, LogLevel.WARN, throwable));
-			FinishTestItemRQ finishRq = buildFinishStepRq(method, ItemStatus.SKIPPED);
-			finishRq.setIssue(Launch.NOT_ISSUE);
-			launch.get().finishTestItem(id, finishRq);
-		});
+		try {
+			Object target = LifecycleHooks.getFieldValue(callable, "val$target");
```

Comment:
The `callable` objects for instance methods always include a reference to the test class instance. The prototype implementation to produce these closures is found in `FrameworkMethod.invokeExplosively`
https://github.com/junit-team/junit4/blob/9ad61c6bf757be8d8968fd5977ab3ae15b0c5aba/src/main/java/org/junit/runners/model/FrameworkMethod.java#L54